home02.js

```diff
@@ -0,0 +1,153 @@
+//Задание 1
+var a = 1, b = 1, c, d;
+c = ++a; alert (c);         //2 - 'c' присваиваетя значение увеличиченной на единицу 'a'
+d = b++; alert (d);         //1 - 'd' присваиваетя значение 'b', затем 'b' увеличивается на единицу
+c = (2 + ++a); alert (c);   //5 - 'a' увеличивается еще на единицу и прибавляется 2
+d = (2 + b++); alert (d);   //4 - к двойке прибавляется 'b', затем 'b' увеличивается на единицу
+alert(a);                   
+alert(b);                   //в итоге и 'a' и 'b' равны 3
+
+//Задание 2
+var a = 2;
+var x = 1 + (a *= 2);
+
+Ответ: x = 5.
+
+//Задание 3
+var a = +prompt('Введите первое целое число');
+var b = +prompt('Введите второе целое число');
+
+if(a >= 0 && b>=0) {
```

Comment:
Не забывайте о пробелах между операторами и операндами
```suggestion
if(a >= 0 && b >= 0) {
```